apps/condo/pages/ticket/pdf.tsx

```diff
@@ -0,0 +1,516 @@
+import { Typography, notification } from 'antd'
+import dynamic from 'next/dynamic'
+import { useRouter } from 'next/router'
+import get from 'lodash/get'
+import { IntlShape } from '@formatjs/intl'
+import dayjs from 'dayjs'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Ticket as ITicket, TicketComment as ITicketComment } from '@app/condo/schema'
+
+import { PageContent } from '@condo/domains/common/components/containers/BaseLayout'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { HtmlToPdf } from '@condo/domains/common/utils/htmlToPdf'
+import { Ticket, TicketComment } from '@condo/domains/ticket/utils/clientSchema'
+import { ParametersType } from '@condo/domains/ticket/hooks/useTicketExportToPdfTask'
+import { getFullAddressByTicket } from '@condo/domains/ticket/utils/helpers'
+import { MAX_TICKET_BLANKS_EXPORT } from '@condo/domains/ticket/constants/export'
+
+
+type BlockRefsType = {
+    blocksRefs: React.MutableRefObject<HTMLDivElement>[]
+    headerRef: React.MutableRefObject<HTMLDivElement>
+    footerRef: React.MutableRefObject<HTMLDivElement>
+}
+
+type TextStyleKeys = 'MAIN_TITLE' | 'SECOND_TITLE' | 'SUBTITLE' | 'TEXT_SIZE_14' | 'TEXT_SIZE_12' | 'TEXT_SIZE_13' | 'TEXT_SIZE_11'
+
+
+const resolutionMultiplier = 3
+const BLOCK_COMMON_STYLE: React.CSSProperties = {
+    width: '100%',
+    paddingLeft: 24 * resolutionMultiplier,
+    paddingRight: 24 * resolutionMultiplier,
+}
+const TEXT_STYLES: Record<TextStyleKeys, React.CSSProperties> = {
+    MAIN_TITLE: { fontSize: 24 * resolutionMultiplier, lineHeight: `${32 * resolutionMultiplier}px`, fontWeight: 700 },
+    SECOND_TITLE: { fontSize: 16 * resolutionMultiplier, lineHeight: `${24 * resolutionMultiplier}px`, fontWeight: 600 },
+    TEXT_SIZE_11: { fontSize: 11 * resolutionMultiplier, lineHeight: `${15 * resolutionMultiplier}px`, fontWeight: 600 },
+    SUBTITLE: { fontSize: 11 * resolutionMultiplier, lineHeight: `${20 * resolutionMultiplier}px`, fontWeight: 600, color: 'grey' },
+    TEXT_SIZE_12: { fontSize: 12 * resolutionMultiplier, lineHeight: `${18 * resolutionMultiplier}px`, fontWeight: 400 },
+    TEXT_SIZE_13: { fontSize: 13 * resolutionMultiplier, lineHeight: `${20 * resolutionMultiplier}px`, fontWeight: 400 },
+    TEXT_SIZE_14: { fontSize: 14 * resolutionMultiplier, lineHeight: `${22 * resolutionMultiplier}px`, fontWeight: 400 },
+}
+
+
+const useQueryParams = (): Record<string, any> => {
+    const router = useRouter()
+    const { query } = router
+    const parsed = {}
+
+    for (const key in query) {
+        parsed[key] = JSON.parse((query as Record<string, string>)[key])
+    }
+
+    return parsed
+}
+
+const SplitBlock = forwardRef<HTMLDivElement>((props, ref) => {
+    const view = (
+        <div style={{ width: 1 * resolutionMultiplier, height: 934 * resolutionMultiplier, backgroundColor: '#E1E5ED' }} ref={ref}></div>
```

Comment:
const style

Reply:
use color from `colors`

Reply:
Some colors are not in the palette. Defined them into constants